### PR TITLE
chore: update side panel handle bar styles to specs

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

**Review by HQFOX — APPROVED** [review]

File: packages/pentaho/src/Canvas/SidePanel/SidePanel.styles.tsx [hunk 1]
```diff
@@ -50,6 +50,7 @@ export const { staticClasses, useClasses } = createClasses(
       },
       "&:hover": {
         cursor: "pointer",
+        backgroundColor: `color-mix(in srgb, ${theme.palette.blue[600]} 10%, transparent)`,
```
Comment (zettca) [inline packages/pentaho/src/Canvas/SidePanel/SidePanel.styles.tsx:53]: We're slowly re-implementing all of `HvIconButton`/`HvButton` unnecessarily. We should reuse `HvIconButton` for the icon

File: packages/pentaho/src/Canvas/SidePanel/SidePanel.tsx [hunk 2]
```diff
@@ -151,7 +152,15 @@ export const HvCanvasSidePanel = forwardRef<
...
-          {open ? <Start color={["primary"]} /> : <End color={["primary"]} />}
+          {open ? (
+            <HvTooltip title={labels.close}>
+              <Start color={["primary"]} />
+            </HvTooltip>
+          ) : (
+            <HvTooltip title={labels.open}>
+              <End color={["primary"]} />
+            </HvTooltip>
+          )}
```
Comment (zettca) [inline packages/pentaho/src/Canvas/SidePanel/SidePanel.tsx:163]: We should be able to replace all of this with:

```tsx
<HvIconButton
  variant="primaryGhost"
  title={open ? labels.close : labels.open}
  onClick={handleTogglePanel}
  className={classes.handle}
>
  <End style={{ rotate: open ? "180deg" : undefined }} />
</HvIconButton>
```

and remove many of the styles too

Comment (plagoa) [reply packages/pentaho/src/Canvas/SidePanel/SidePanel.tsx]: Yeah, I updated I removed this clutter.

Comment (zettca) [reply packages/pentaho/src/Canvas/SidePanel/SidePanel.tsx]: there's still quite a few redundant/duplicated styles BTW

File: packages/pentaho/src/Canvas/SidePanel/SidePanel.styles.tsx (hunk 1 above)
Comment (plagoa) [reply packages/pentaho/src/Canvas/SidePanel/SidePanel.styles.tsx]: Yeah, I removed the redundant styles.

File: packages/pentaho/src/Canvas/SidePanel/SidePanel.tsx (hunk 2 above)
Comment (plagoa) [reply packages/pentaho/src/Canvas/SidePanel/SidePanel.tsx]: Yeah, I removed the redundant styles.

**Review by zettca — APPROVED** [review]

**Pull request merged**